### PR TITLE
Update to v2.0.9

### PR DESCRIPTION
**Pull request by ramkrsna**
- Update to version 2.0.9
- the update fixes VDO device persistence when the devices are rebooted.

### CODE REVIEW

File: sources [hunk 1]
```diff
@@ -1 +1,2 @@
-SHA512 (gdeploy-2.0.8.tar.gz) = de3e8dee3c233e9928c0e2b4f3c59fb3809f049c96a4eb93c21696d3017513890919b06e4409ff6a593117881f4c43ec2e2f84e3c1c5592c49d501ec437cca4e
+SHA512 (gdeploy-2.0.9.tar.gz) = 51863aa3feea67b96db9a0de5a8b94c4d32692e7c2de9505cc51b9ddcaa35663b8f950c1e66df2358d95a86609f374a15c50e7cf628209523293814dfb787c4c
+
```
Comment (nixpanic) [inline sources]: When I apply this, I get the following warning:

```
$ curl https://patch-diff.githubusercontent.com/raw/CentOS-Storage-SIG/gdeploy/pull/3.diff | git apply 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1120    0  1120    0     0   5544      0 --:--:-- --:--:-- --:--:--  5544
<stdin>:34: new blank line at EOF.
+
warning: 1 line adds whitespace errors.
```

File: gdeploy.spec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 %global gdeploytemp %{_datadir}/gdeploy
 
 Name:           gdeploy
-Version:        2.0.8
+Version:        2.0.9
```
Comment (nixpanic) [inline gdeploy.spec:6]: There is no matchin tag in the deploy github repository yet?

```
$ spectool -g -S gdeploy.spec 
Getting https://github.com/gluster/gdeploy/archive/v2.0.9.tar.gz#/gdeploy-2.0.9.tar.gz to ./gdeploy-2.0.9.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   123    0   123    0     0    266      0 --:--:-- --:--:-- --:--:--   266
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
```

Comment (ramkrsna) [reply gdeploy.spec:6]: @nixpanic I have fixed the tags in origin, thanks for spotting it.

**Comment by nixpanic** [comment]
@ramkrsna the package is available in the test repositories. Can you validate the package and let me know when I can mark it as release? The following should work on a clean CentOS environment:

```
yum install centos-release-gluster
yum install --enablerepo='centos-gluster*-test" install gdeploy
```

Any testing should not result in missing dependencies or otherwise broken functionality.

**Pull request closed without merging**